package.json

```diff
@@ -33,7 +36,8 @@
   "jest": {
     "roots": [
       "app/javascript/components"
-    ]
+    ],
+    "testEnvironment": "jsdom"
   },
   "scripts": {
     "test": "jest --coverage"
```

Comment:
In the future, you can separate your scripts such that `yarn test:coverage` would run this command and `yarn test` runs your jest... also you can add a `yarn test:watch` adds the `--watchAll` param... you'll see this pattern a lot. When you start having projects go through CI/CD process you'll have specific test scripts.